### ISSUE TITLE
Importing multiple services uses settings of the last selected one

### ISSUE THREAD

**Issue by WilliamRoxit** [statping-ng/statping-ng#219]
If I import 6 different services (from a export file), then I will have 6 instances but they all have the name and settings of the lastly selected service on the importscreen.

**Comment by RoyTakanen** [comment]
Can confirm this too.

Import page

<img width="685" alt="image" src="https://user-images.githubusercontent.com/52327670/222404842-ecdb6205-ae6e-4c06-9897-b6c48f2329fd.png">
 

Result of the import

<img width="959" alt="image" src="https://user-images.githubusercontent.com/52327670/222404610-ff13e8f0-4c39-4093-92a2-3eb78df1c92f.png">

It is strange that the HTTP response returns the correct services though

<img width="1438" alt="image" src="https://user-images.githubusercontent.com/52327670/222405292-28fb2cbe-089a-4bab-8105-3971ec14d5ae.png">

**Comment by WilliamRoxit** [comment]
I actually didn't look into th ehttp response at all. But as you can see, all ID's are 0; so when importing multiple services at once, there are no ID's generated for each service. That is probably also the reason why you can't delete them anymore :)

**Comment by RoyTakanen** [comment]
Yeah that seems to be the case. I went to check the code and found the following: https://github.com/statping-ng/statping-ng/blob/dev/handlers/dashboard.go#L236. Git blame also [reports](https://github.com/statping-ng/statping-ng/blame/7abc4e8203dcb69f52a8f6fc4dbd8a67c476095a/handlers/dashboard.go#L236) reports that the commit name is `export HTTP test`. I think that the line should probably be removed :D. It might have been some debugging stuff that was used when the [tests for export](https://github.com/statping-ng/statping-ng/commit/79918d025d4fbbde9cc005aa561b0e2e9d5efc53) were written, but I do not really know.

**Comment by WilliamRoxit** [comment]
Interesting find. But then it also applies to users and groups, since there the ID is also forced to 0 (same file).

**Comment by kroese** [comment]
Just ran into this issue. Would be nice if it would be fixed.
